libraries/src/test/java/com/baeldung/mapdb/SortedTableMapUnitTest.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.mapdb;
+
+import org.junit.Test;
+import org.mapdb.Serializer;
+import org.mapdb.SortedTableMap;
+import org.mapdb.volume.MappedFileVol;
+import org.mapdb.volume.Volume;
+
+import static junit.framework.Assert.assertEquals;
+
+public class SortedTableMapUnitTest {
+
+  @Test
+  public void givenValidSortedTableMapSetup_whenQueried_checkValuesCorrect() {
+
+    final String VOLUME_LOCATION = "sortedTableMapVol.db";
+
+    //create memory mapped volume, readonly false
+    Volume vol = MappedFileVol.FACTORY.makeVolume(VOLUME_LOCATION, false);
+
+    //create sink to feed the map with data
+
+    SortedTableMap.Sink<Integer, String> sink =
+        SortedTableMap.create(
+            vol,
+            Serializer.INTEGER,
+            Serializer.STRING
+        ).createFromSink();
+
+    //add content
+    for(int i = 0; i < 100; i++){
+      sink.put(i, "Value " + String.valueOf(i));
```

Comment:
It might be more correct to use the `Integer` method to create the string - https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html#toString-int-

Reply:
Changed `String.valueOf(i)` to `Integer.toString(i)`